@xen-orchestra/web-core/lib/components/tab/VtsTabItem.vue

```diff
@@ -0,0 +1,84 @@
+<!-- v2 -->
+<template>
+  <component :is="tag" :class="classNames" class="vts-tab-item">
+    <slot />
+  </component>
+</template>
+
+<script lang="ts" setup>
+import { useContext } from '@core/composables/context.composable'
+import { DisabledContext } from '@core/context'
+import { useUiStore } from '@core/stores/ui.store'
+import { storeToRefs } from 'pinia'
+import { computed } from 'vue'
+
+const props = withDefaults(
+  defineProps<{
+    disabled?: boolean
+    selected?: boolean
+    tag?: string
+  }>(),
+  { tag: 'span', disabled: undefined }
```

Comment:
Yes, it is to prevent `undefined` to `false` conversion by Vue. In this case it is necessary because the `disabled` prop is used with `useContext()` on line 30. `selected` is not used in the context, so there is no need to do the same.
More information [here](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/web-core/docs/composables/context.composable.md#caveats-boolean-props)